test/User-test.js

```diff
@@ -33,9 +40,23 @@ describe('User', () => {
   it('Should start with no booking', () => {
     expect(user1.userBookings).to.deep.equal([]);
     expect(user2.userBookings).to.deep.equal([]);
-  })
+  });
 
+  it('should dafault to 0 booking cost', () => {
+    expect(user1.bookingCost).to.equal(0);
+  });
 
+  it('Should display any bookings user has done', () => {
+    user1.retrieveRoomBookings(bookingsData);
+    expect(user1.userBookings.length).to.deep.equal(1)
+  });
```

Comment:
It'd strengthen this test to add another `expect` statement that checks the actual contents of the user bookings. Length of 1 just says that _something_ is there, not necessarily what it is.